src/main/java/net/openhft/chronicle/queue/ChronicleQueue.java

```diff
@@ -196,9 +196,7 @@ default LongValue indexForId(String id) {
      * this method every time an appender is needed.
      *
      * @return Returns a ExcerptAppender for this ChronicleQueue that is local to the current Thread
-     * @deprecated It is recommended to use {@link #createAppender()} instead
      */
-    @Deprecated(/* To be removed in x.27 */)
```

Comment:
needs broader discussion before un-deprecating

Reply:
We decided that making services fail on deprecation was more important.